app/custom/templates/blog-single/index.js

```diff
@@ -0,0 +1,3 @@
+/* Add requires here for all directives referenced by the home template */
+require( './../../organisms/header/directive' );
```

Comment:
No @adamf321, nowadays patternlab only includes the custom components in the patterns page, but it does not generates the requires for them.